app/controllers/content_items_controller.rb

```diff
@@ -5,6 +5,6 @@ def index
   end
 
   def show
-    head :ok
```

Comment:
Oh! I didn't think about those. I assume I add test to check the status when the `show` method is added, and then the other tests when create the `@content_item` instance variable?